session/service.go

```diff
@@ -0,0 +1,121 @@
+package session
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb/v2"
+	"github.com/influxdata/influxdb/v2/rand"
+	"github.com/influxdata/influxdb/v2/snowflake"
+)
+
+type Service struct {
+	store         *Storage
+	userService   influxdb.UserService
+	urmService    influxdb.UserResourceMappingService
+	sessionLength time.Duration
+
+	idGen    influxdb.IDGenerator
+	tokenGen influxdb.TokenGenerator
+}
+
+func NewService(store *Storage, userService influxdb.UserService, urmService influxdb.UserResourceMappingService, sessionLength time.Duration) *Service {
```

Comment:
Lots of these things don't have any function signature docs. Maybe I just am a pedant after years of golint screaming at me to add them, but what is our policy for docs? (I genuinely don't know) I probably would at the minimum give the core service types and functions a little explanation.